### PR TITLE
feat: APPS-2221_add-related-items-metadata-field

### PR DESCRIPTION
**Pull request by jendiamond**

### CODE REVIEW

File: app/models/solr_document.rb [hunk 1]
```diff
@@ -155,7 +155,7 @@ def host
   end
 
   def iiif_manifest_url
-    self[:iiif_manifest_url_ssi] || "/concern/works/#{id}/manifest"
+    self[:iiif_manifest_url_ssi] || "https://iiif.library.ucla.edu/ark%3A%2F21198%2Fzz00090p17/manifest"
```
Comment (sourcefilter) [inline app/models/solr_document.rb:158]: We can't actually merge this, but it's great that it fixed the issue!

Comment (sourcefilter) [reply app/models/solr_document.rb:158]: Probably best to just try `self[:iiif_manifest_url_ssi]` without the `|| ...` bit?

**Review by sourcefilter — CHANGES_REQUESTED** [review]
😎 , figured out the issue! Just need to find a solution we can use in production! Hopefully we can just use blank manifest urls, but if not we can make one that specifically says something like "missing image"

File: app/models/solr_document.rb (hunk 1 above)
Comment (jendiamond) [reply app/models/solr_document.rb:158]: substituted:
"https://p-w-dl-viewer01.library.ucla.edu/#?manifest=#{CGI.escape(http_sub(iiif_manifest_url.to_s))}"

**Review by sourcefilter — APPROVED** [review]

**Pull request merged**